environments/staging/inventory.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+all:
+  hosts:
+    185.189.221.157: null
```

Comment:
Since this is specific to us: when we open source the ansible files we simply leave out the inventory files?

Reply:
Well, I mean they are already open source ;-) - at least as in they are publicly available here. Leaving out the inventory is one option. The better option would pobably be bundling them as a a collection to put on ansible-galaxy.

Reply:
I agree that the file should be left out by default, since the the ansible repo is directed at interested people deploying their own (not our) palava. We could then just use a non-default branch with our inventory, or organize it differently.

If we choose to not include the inventory, we should still have an "example" inventory file and document it int the README, that you have to copy and adjust it.